hardemu/screen.js

```diff
@@ -721,9 +721,9 @@ function VideoChip(Zeal, PIO, scale) {
             video_cfg.height = 480;
             video_cfg.base_scale = 1;
         }
-        const scale = video_cfg.base_scale * video_cfg.view_scale;
-        canvas.style.transform = `scale(${scale})`;
-        canvas_layer1.style.transform = `scale(${scale})`;
+        // const scale = video_cfg.base_scale * video_cfg.view_scale;
```

Comment:
This is used to make the `320x240` screen canvas the same size as the `640x480` one. Removing this would result in a smaller canvas when using `320x240` resolution

Reply:
I override the width/height with CSS and the outcome is identical… the problem I ram into is that some other changes I made cause the canvas to offset itself when switching video mode… this is a fix to that

it should work just the same - tested on my end